cms/tests/test_apphooks.py

```diff
@@ -32,7 +32,7 @@
 APP_MODULE = "cms.test_utils.project.sampleapp.cms_apps"
 
 
-class ApphooksTestCase(CMSTestCase):
+class ApphooksTestCase(ClearURLs, CMSTestCase):
```

Comment:
Could you create a tech debt ticket with steps to reproduce and any knowledge you have from working on this issue? I might try to get to the bottom of it.